### PR TITLE
Feature/pt 94

### PR DESCRIPTION
**Pull request by AdamSarach**

### CODE REVIEW

File: src/components/DashboardSiderbar.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import PropertyGrid from './PropertyGrid';
 import MapPanel from './MapPanel';
 import SidebarMenu from './SidebarMenu';
 import LanguageSwitch from './LanguageTest';
+import NewReport from "./NewReport";
```
Comment (100hel) [inline src/components/DashboardSiderbar.jsx:7]: As far as I can see we use a single quotation for the imports. We should configure the ESLint.

File: src/components/mapComponents/NewReportMarker.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {Marker} from 'react-leaflet';
+import {getReportIcon} from '../../utilities/utilities';
+
+const ReportMarkers = ({newReportCoordinates}) => {
+  const {lat, lon} = newReportCoordinates || {};
+  return lat && lon && <Marker position={[lat, lon]} icon={getReportIcon()} />;
```
[the remark this answers is not in the document]
Comment (AdamSarach) [reply src/components/mapComponents/NewReportMarker.jsx:7]: I think I see your point of view

File: src/utilities/utilities.js [hunk 3]
```diff
@@ -45,6 +46,17 @@ const getBusStopIcon = busStopPropeties => {
...
+const getReportIcon = () => {
+    return new Icon({
+        iconUrl: iconOrange,
+        shadowUrl: iconShadow,
+        iconSize: [30, 55],
+        iconAnchor: [15, 54],
+        shadowSize: [68, 60],
+        shadowAnchor: [-10, 55],
+    });
+};
```
[the remark this answers is not in the document]
Comment (AdamSarach) [reply src/utilities/utilities.js:58]: Formatting is done by prettier. I will take a look into configuring ESlint in another task

File: src/components/mapComponents/NewReportMarker.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {Marker} from 'react-leaflet';
+import {getReportIcon} from '../../utilities/utilities';
+
+const ReportMarkers = ({newReportCoordinates}) => {
+  const {lat, lon} = newReportCoordinates || {};
+  return lat !== null && lon !== null && <Marker position={[lat, lon]} icon={getReportIcon()} />;
```
Comment (100hel) [inline src/components/mapComponents/NewReportMarker.jsx:7]: Maybe `(typeof lat === 'number')`?

**Review by 100hel — APPROVED** [review]

**Pull request merged**